omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/ProgramEnrollmentResourceTest.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2;
+
+import org.junit.Before;
+import org.openmrs.PatientProgram;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+
+@org.springframework.test.context.ContextConfiguration(locations = { "classpath:TestingApplicationContext.xml" }, inheritLocations = true)
+public class ProgramEnrollmentResourceTest extends BaseDelegatingResourceTest<ProgramEnrollmentResource, PatientProgram> {
+	
+	protected static final String PROGRAM_ATTRIBUTES_XML = "org/openmrs/api/include/ProgramAttributesDataset.xml";
+	
+	@Before
+	public void before() throws Exception {
+		executeDataSet("programEnrollmentDataSet.xml");
+	}
+	
+	@Override
+	public PatientProgram newObject() {
+		return Context.getProgramWorkflowService().getPatientProgramByUuid(getUuidProperty());
+	}
+	
+	@Override
+	public void validateDefaultRepresentation() throws Exception {
+		super.validateDefaultRepresentation();
+		//assertPropPresent("attributes");
```

Comment:
Did you intentionally comment this out?

Reply:
It's from the bahmni source, I'm still making the changes.

Reply:
Let us just remove it